src/kbmod/analysis/plot_results.py

```diff
@@ -56,7 +63,12 @@ def plot_stamps(stamps, fig=None, columns=3):
 
         for i in range(num_stamps):
             ax = fig.add_subplot(num_rows, columns, i + 1)
-            ResultsVisualizer.plot_single_stamp(stamps[i], axes=ax)
+            norm = (
+                ImageNormalize(stamps[i], interval=ZScaleInterval(), stretch=AsinhStretch())
+                if normalize
+                else None
+            )
```

Comment:
Can't this just be 

```
for i, stamp in enumerate(stamps):
    ax = fig.add_subplot(n_rows, n_cols, i+1)
    ax.set_title(f"Time {i}")
    norm = None
    if normalize:
        norm = ImageNormalize(stamp, interval=ZScaleInterval(), stretch=AsinhStretch())
    ResultsVisualizer.plot_single_stamp(stamp, axes=ax, norm=norm)
```

https://github.com/dirac-institute/kbmod/issues/495 doesn't specify whether or not normalization should happen for each image individually or whether they should all be normalized to the same baseline. If it's the latter, then the norm needs to be shared by all the images. 

We could calculate it for the first stamp in the list, or some kind of statistic across the stamps (like a mean, median, min, max or something), we could also let users supply their own `norm` since `plot_single_stamp` allows for it already. Addressing this would also probably fix the awkward `norm` if conditional too. 

This is fine as-is, but we should probably raise this as a point(s) of discussion in https://github.com/dirac-institute/kbmod/issues/495 or https://github.com/dirac-institute/kbmod/issues/523

Reply:
Changed it to the more readable version you suggested. We can hold off merging this until we address it in either #495 or #523 ? Alternatively, if you prefer to merge as is and then remove later then that also works (but I cannot merge as I'm not a contributor).

Reply:
I'm ok with merging this now, the core functionality would remain the same, it's really just the func signatures and module reorganization that we need a better plan for I think. I added you to contributors, hopefully this becomes less of a blocker for you now. We can't use the github Runners for the CI because we need GPUs, so we had to deploy our own; please don't mine bitcoins on it.